### ISSUE TITLE
problem when using with browserify-ngannotate

### ISSUE THREAD

**Issue by elijahwww** [majgis/ngify#18]
I ran into a problem when using ngify along with browserify-ngannotate transform. That one uses `/*@ngInject */` annotations, and looks like `@ng` gets tripped up by it.

  ```
var b = browserify(options)
   .transform(require('ngify'))
   .transform(require('browserify-ngannotate'))
```

```
app.directive('bla', /* @ngInject */ function (serviceA){
	return {
		templateUrl: require('../templates/test.html'),
	};
});
```
Also, how do I programmatically configure this transformer? (no package.json)

**Comment by majgis** [comment]
I will look into the `@ngInject` issue.

In regard to programmatically configuring it, the transform method takes a [second `opts` parameter](https://github.com/substack/node-browserify#btransformtr-opts)

I created a new issue with an example here: https://github.com/majgis/ngify/issues/19

**Comment by elijahwww** [comment]
Excellent, thank you!

**Comment by majgis** [comment]
I'm going to add a permanent fix for this issue, but I think you can immediately resolve the problem by adding ngify second:

    var b = browserify(options)
       .transform(require('browserify-ngannotate'))
       .transform(require('ngify'))

**Comment by majgis** [comment]
@elijahwww This issue should be resolved in release v1.3.3

Please let me know if you are still experiencing issues.

**Comment by elijahwww** [comment]
This is working! Thank you.
